pymc3/sampling.py

```diff
@@ -280,6 +283,37 @@ def sample(draws=500, step=None, init='auto', n_init=200000, start=None,
     return sample_func(**sample_args)[discard:]
 
 
+def _check_start_shape(model, start):
+    e = ''
+    for var in model.vars:
+        if np.shape(start):
```

Comment:
Yes, that might be more clear. Basically I saw that this was how `_make_parallel` checked to see if `arg` was array-like, so I used the same expression. But I'm happy to change it to something more clear.